schema/bom-1.6.proto

```diff
@@ -436,19 +436,19 @@ enum LicensingTypeEnum {
 }
 
 message Metadata {
-  // The date and time (timestamp) when the document was created.
+  // The date and time (timestamp) when the CycloneDX document was created.
   optional google.protobuf.Timestamp timestamp = 1;
-  // The tool(s) used in the creation of the BOM.
+  // The tool(s) used in the creation of the Cyclonedx document.
   optional Tool tools = 2;
-  // The person(s) who created the BOM. Authors are common in BOMs created through manual processes. BOMs created through automated means may not have authors.
+  // The person(s) who created the CycloneDX document. Authors are common in documents created through manual processes. Documents created through automated means may not have authors. This may be different from the author(s) of the component that the the CycloneDX document describes.
```

Comment:
```suggestion
  // The person(s) who created the CycloneDX document. Authors are common in documents created through manual processes. Documents created through automated means may not have authors. This may be different from the author(s) of the component the CycloneDX document describes.
```